Makefile

```diff
@@ -23,10 +23,12 @@ endef # cddl_check_template
 CORIM_FRAGS := concise-rim.cddl
 CORIM_FRAGS += signed-corim.cddl
 CORIM_FRAGS += unsigned-corim.cddl
-CORIM_FRAGS += concise-mid-tag.cddl
+#CORIM_FRAGS += concise-mid-tag.cddl
```

Comment:
Perhaps we can later tidy this up  once we agree that we do not want old files in here!

Reply:
sure